Documentation/adding-new-repository.md

```diff
@@ -29,6 +30,9 @@ To create the project in GitLab:
 2. Log into GitHub as `mender-test-bot`, credentials are in `mystiko`
 3. Log into GitLab, using OAuth from GitHub. If asked for the token, use the one in `mystiko`
 4. New project >  Run CI/CD for external repository
+5. Set the branch protections, for staging, feature, and release branches (see picture below):
+
+![branch-protections](gitlab-branch-protections.png)
```

Comment:
This would be better in `GitHub: branch protections` section. We can rename the section to be `Branch protections` and then explicitly say that they have to be added both in GitHub and in GitLab.

Reply:
I am sorry, it makes sense. fixed.